src/frontend/src/components/public/PIAFormTabs/intake/index.tsx

```diff
@@ -67,30 +82,43 @@ export const PIAFormIntake = () => {
             <div className="row">
               <InputText
                 label="Title"
+                fieldMandatoryLabel="(required)"
```

Comment:
isn't "required" label consistent for all fields? So, why send from all InputText usages?